python-avd/schema_tools/metaschema/meta_schema_model.py

```diff
@@ -277,7 +277,7 @@ class ConvertType(str, Enum):
     """Default value"""
     valid_values: list[bool] | None = None
     """List of valid values"""
-    dynamic_valid_values: str | None = None
+    dynamic_valid_values: list[str] | None = None
     """
     Path to variable under the parent dictionary containing valid values.
```

Comment:
Should the description change? It is now a list of paths?

Reply:
fixed